content/ja/handbook/engineering/infrastructure-platforms/environments/staging-ref.md

```
@@ -0,0 +1,204 @@
+
+### 環境情報
+
+- [Geo](https://docs.gitlab.com/ee/administration/geo/) は以下の設定で Staging Ref にセットアップされています：
+  - Staging Ref US サイト - *プライマリ* - [3k クラウドネイティブハイブリッドリファレンスアーキテクチャ](https://docs.gitlab.com/ee/administration/reference_architectures/3k_users.html#cloud-native-hybrid-reference-architecture-with-helm-charts-alternative) 環境 - ステートレスコンポーネント（Webservice、Sidekiq、NGINX）は Google Kubernetes Engine クラスターにデプロイされ、残りのステートフルコンポーネントは GCP 仮想マシンにインストールされています
+  - Staging Ref EU サイト - *セカンダリ* - [3k リファレンスアーキテクチャ](https://docs.gitlab.com/ee/administration/reference_architectures/3k_users.html) 完全な Omnibus 環境
+- [GitLab Environment Toolkit (GET)](https://gitlab.com/gitlab-org/gitlab-environment-toolkit) と [Deployer](https://ops.gitlab.net/gitlab-com/gl-infra/deployer) でデプロイ
+- SSL 証明書は [Let's Encrypt](https://letsencrypt.org/) で自動化
+- [Google OAuth](https://docs.gitlab.com/ee/integration/google.html) で GitLab チームメンバーの環境アクセスを提供
+- [送信メール](https://docs.gitlab.com/charts/charts/globals.html#outgoing-email) は Mailgun で設定
+- [高度な検索](https://docs.gitlab.com/ee/user/search/advanced_search.html) は Elasticsearch と [GET](https://gitlab.com/gitlab-org/gitlab-environment-toolkit/-/blob/main/docs/environment_advanced.md#advanced-search-with-elasticsearch) で設定
+- デフォルトで [Free 有料プラン](#upgrade-paid-plans) の Ultimate ライセンス
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)